arrow/buffer/bitmap.mojo

```diff
@@ -78,9 +78,19 @@ struct Bitmap(StringableRaising):
         var bitmask: UInt8 = 1 << (index % 8)
         return ((self._buffer[byte_index] & bitmask)).__bool__()
 
-    fn __getitem__(self, index: Int) raises -> Bool:
-        if index < 0 or index >= self.length:
-            raise Error("index out of range for Bitmap")
+    fn __getitem__(self, index: Int) -> Bool:
+        """Get the value at the index. If the index is out of bounds, then
+        return False.
+
+        Args:
+            index: The index [0, len(self) -1].
+
+        Returns:
+            The value.
+        """
+
+        if not (0 <= index < self.length):
+            return False
```

Comment:
Please change this back, a design change like this should be submitted as a separate PR instead of being added to this one!